src/app/(topics)/ai/autoregressive/Autoregressive.module.css

```
@@ -0,0 +1,260 @@
+  border: 1px solid var(--color-border);
+  border-radius: var(--radius-lg);
+  background: var(--color-surface);
+}
+
+/* 마크다운 문단이 <p> 로 들어오므로 래퍼는 div 다 (§4.6). */
+.sectionLead {
+  margin-bottom: var(--space-md);
+  color: var(--color-muted);
+}
+
+.sectionLead :global(p + p) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)